Build fails with the recent schema from Apollo

Hi,

The rover build starts failing sometime in the last four days (last time we've attempted on building it was 4 days ago and it was successful, and today it fails).

This is the error:
```
error[E0063]: missing field `excludedOperationNames` in initializer of `graph_check_mutation::HistoricQueryParameters`
  --> crates/rover-client/src/operations/graph/check/types.rs:34:9
   |
34 |         Self {
   |         ^^^^ missing `excludedOperationNames`

error[E0063]: missing field `excludedOperationNames` in initializer of `subgraph_check_mutation::HistoricQueryParameters`
  --> crates/rover-client/src/operations/subgraph/check/types.rs:71:9
   |
71 |         Self {
   |         ^^^^ missing `excludedOperationNames`

For more information about this error, try `rustc --explain E0063`.
error: could not compile `rover-client` due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
error: build failed
```

You can see it failing in your CI:
https://app.circleci.com/pipelines/github/apollographql/rover/618/workflows/e0a64815-3694-4570-be6b-12c5a36f4eb5/jobs/3702

Not sure if this something that's broken in the Apollo schema (and if so, how to report it) or something that's missing in Rover to support this change in the schema.

--- a/crates/rover-client/src/operations/graph/check/types.rs
+++ b/crates/rover-client/src/operations/graph/check/types.rs
@@ -38,6 +38,7 @@ impl From<CheckConfig> for MutationConfig {
             to,
             // we don't support configuring these, but we can't leave them out
             excludedClients: None,
+            excludedOperationNames: None,
             ignoredOperations: None,
             includedVariants: None,
         }

--- a/crates/rover-client/src/operations/subgraph/check/types.rs
+++ b/crates/rover-client/src/operations/subgraph/check/types.rs
@@ -75,6 +75,7 @@ impl From<CheckConfig> for MutationConfig {
             to,
             // we don't support configuring these, but we can't leave them out
             excludedClients: None,
+            excludedOperationNames: None,
             ignoredOperations: None,
             includedVariants: None,
         }

--- a/src/utils/env.rs
+++ b/src/utils/env.rs
@@ -67,7 +67,7 @@ impl RoverEnv {
             value.to_string()
         };
 
-        format!("environment variable ${} = {}", key.to_string(), value)
+        format!("environment variable ${} = {}", key, value)
     }
 
     /// sets an environment variable to a value

